### PR TITLE
Add Trino Fest preview blog

### PR DESCRIPTION
**Pull request by colebow**

### CODE REVIEW

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/announcement-banner.png
+show_pagenav: true
+---
+
+Trino Fest is drawing ever closer, and Commander Bun Bun has been hard at work
+behind the scenes arranging the schedule and making sure that Trino's trip to
+Boston is going to be a great one. In case you missed it,
+[we announced Trino Fest]({{site.baseurl}}{% post_url 2024-02-20-announcing-trino-fest-2024 %})
+a couple months ago, and if you have missed it, make sure to go register to
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:14]: remove the duplication of "missed it" somehow

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+attend! All our speakers will be in person in downtown Boston on the 13th of
+June, with plenty of opportunities for networking and a 2-hour happy hour at the
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:16]:
```suggestion
June, with plenty of opportunities for networking and a happy hour event at the
```

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+end of the day. But if you can't make the trip up to enjoy the lovely New
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:17]:
```suggestion
end of the day. But if you can't make the trip to enjoy the lovely New
```

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+join us virtually.
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-orange" href="http://www.starburst.io/info/trino-fest-2024?utm_medium=trino&utm_source=website&utm_campaign=Global-FY25-Q2-EV-Trino-Fest-2024&utm_content=Blog-2">
+        Register to attend!
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+Still on the fence, though? Read on for a sneak peek at our speaker lineup and
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:28]: not just a sneak peak...

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+previews of their talks. Companies being represented include names like Apple 
+and Amazon, so you know it's going to be good.
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:30]: Do we need to add Microsoft? I think yes .. given its their first time. Or leave the name calling out?

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+<!--more-->
+
+## Data lakes
+
+It's no secret that using Trino as part of your data lake has become one of its
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:36]: well... it was the major use case from the start .. so either reframe to lakehouses .. or change altogether

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+major use cases in the past few years. We're excited to say that at Trino Fest,
+we'll have representation for each of the modern big three: Iceberg, Delta Lake,
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:38]:
```suggestion
we'll have representation for each of the modern big three table formats: Iceberg, Delta Lake,
```

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+and Hudi.
+
+### Iceberg
+
+Iceberg will be covered twice: Amogh Jahagirdar from Tabular will be diving into
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:43]:
```suggestion
Apache Iceberg will be covered twice: Amogh Jahagirdar from Tabular will be diving into
```

Comment (mosabua) [reply _posts/2024-04-15-trino-fest-2024-approaches.md:43]: Here or somewhere else .. also ... I think we should link to all the companies and projects here and in following sections

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+the world of Iceberg views and how they can be leveraged to coordinate across
+different query languages and dialects; Amit Gilad from Cloudinary will be
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:45]: Should the two talks just be separate bullet points?

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+covering the story of migrating out of Snowflake to the wonderful world of open
+table formats and Iceberg.
+
+### Delta Lake
+
+Marius Grama, a Trino maintainer at Starburst, will be going into detail on the
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:51]:
```suggestion
Marius Grama, a Trino contributor at Starburst, will be going into detail on the
```

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+history, development, and improvements to the Delta Lake connector. With time
+travel for the Delta Lake connector landed in Trino 445, it's one of the most
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:53]: link to the other blog post? Also .. it reads a bit weird .. time travel is just a small part of all the improvements and progress he will talk about

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+exciting areas for development in open source Trino, and there's some
+interesting technical stories that Marius is excited to share with the
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:55]:
```suggestion
interesting stories that Marius is excited to share with the
```

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+community.
+
+### Hudi
+
+Rounding out data lakes, Ethan Guo from Onehouse will be diving into Trino's
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:60]: OneHouse ?

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+Hudi connector, giving an update on what's been landed lately to improve
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:61]:
```suggestion
Hudi connector, giving an update on what landed lately to improve
```

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+performance and functionality, and he'll also be giving a preview of what's
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:62]:
```suggestion
performance and functionality.  He'll also give a preview of what's
```

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 16]
```diff
@@ -0,0 +1,124 @@
...
+coming soon. The features are flying in, and if you're a current or prospective
+user of Hudi with Trino, you won't want to miss out.
+
+### Amazon S3
+
+And for all your data lake needs, Dai Ozaki from Amazon will be discussing best
+practices for running Trino on S3 data lakes - no matter which storage system
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:69]:
```suggestion
practices for running Trino with S3 data lakes - no matter which storage system
```

Comment (mosabua) [reply _posts/2024-04-15-trino-fest-2024-approaches.md:69]: or "S3-backed data lakes"

Comment (mosabua) [reply _posts/2024-04-15-trino-fest-2024-approaches.md:69]: Also no matter which table format .. or whatever .. S3 is the storage system ...

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+you're using. It's a great opportunity to learn about how you can scale
+workloads, optimize data layouts, or reduce latency when using S3 for your data
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:71]:
```suggestion
workloads, optimize data layouts, and reduce latency when using S3 for your data
```

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 18]
```diff
@@ -0,0 +1,124 @@
...
+Of course, there's more to Trino than querying data lakes, and there's a wide
+variety of talks to discuss the other activities going on within the Trino
+community.
+
+### Small scale
+
+Ben Jeter at Executive Homes, who gave [a talk at Trino Fest last year]({{site.baseurl}}{% post_url 2023-07-25-trino-fest-2023-datto %})
+while at Datto, is back to discuss running Trino at a more moderate scale than
+that we're used to hearing about in the Trino space. Forget petabytes and
+exabytes, and welcome a tiny cluster querying thousands (not millions) of
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:85]:
```suggestion
exabytes, and welcome a tiny cluster querying thousands, not millions, of
```

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+records that still derives huge value from Trino. It's a great playbook for
+smaller startups and enterprises who still need robust, performant analytics.
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:87]: I think is also a lot about flexibility ..

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 20]
```diff
@@ -0,0 +1,124 @@
...
+
+### Maximizing performance
+
+Jonas Kylling from Dune Analytics will be detailing how they've managed to
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:91]: I think its just Dune - https://dune.com/about

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 21]
```diff
@@ -0,0 +1,124 @@
...
+optimize Trino and squeeze out every ounce of performance to reduce query costs
+and runtimes. That includes leveraging the new Alluxio-based file system
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:93]: Mention that they wrote most of it?

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 22]
```diff
@@ -0,0 +1,124 @@
...
+## <a name="tcc"/> Trino contributor congregation
+
+The day after Trino Fest, we'll also be hosting an in-person meetup for
+Trino contributors and engineers to catch up, discuss the Trino roadmap, and
+engage directly with the maintainers in-person. It's a great opportunity to put
+faces and voices to those GitHub handles, align on the big ideas or tricky PRs
+that have been moving slowly, and find more ways to get involved in Trino
+development. If you're interested in attending, message Manfred Moser or Cole
+Bowden on the [Trino Slack]({{site.url}}/slack.html), and we'll get you added to
+the attendee list and share more details.
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:124]: Need to add a section about the sponsors with the logos at the bottom here .. or maybe higher up.. but somewhere

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 23]
```diff
@@ -0,0 +1,124 @@
...
+Jonas Kylling from Dune Analytics will be detailing how they've managed to
+optimize Trino and squeeze out every ounce of performance to reduce query costs
+and runtimes. That includes leveraging the new Alluxio-based file system
+caching, emulating various cluster sizes to avoid expensive idle cluster time,
+and storing, sampling, and filtering query results to avoid re-executing
+queries.
+
+### Query intelligence
+
+Marton Bod at Apple brings insights to query intelligence. He'll demonstrate how
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:100]:
```suggestion
Marton Bod from Apple brings insights to query intelligence. He'll demonstrate how
```

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2024-04-15-trino-fest-2024-approaches.md (hunk 1 above)
Comment (colebow) [reply _posts/2024-04-15-trino-fest-2024-approaches.md:14]: That's on purpose. It's uh... anaphora. I'll add italics to make it more clear.

File: _posts/2024-04-15-trino-fest-2024-approaches.md (hunk 5 above)
Comment (colebow) [reply _posts/2024-04-15-trino-fest-2024-approaches.md:30]: We don't have confirmation from them yet

File: _posts/2024-04-15-trino-fest-2024-approaches.md (hunk 13 above)
Comment (colebow) [reply _posts/2024-04-15-trino-fest-2024-approaches.md:60]: Onehouse is correct, I believe. Their website has it that way.

File: _posts/2024-04-15-trino-fest-2024-approaches.md (hunk 10 above)
Comment (findinpath) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:51]: maintainer -> contributor

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 24]
```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "A sneak peek of Trino Fest 2024"
```
[the remark this answers is not in the document]
Comment (colebow) [reply _posts/2024-04-15-trino-fest-2024-approaches.md:3]: I think this is the most compelling angle for it?

[the remark this answers is not in the document]
Comment (mosabua) [reply _posts/2024-04-15-trino-fest-2024-approaches.md:3]: ok

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 25]
```diff
@@ -0,0 +1,157 @@
...
+author: "Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/announcement-banner.png
+show_pagenav: true
+---
+
+Trino Fest is drawing ever closer, and Commander Bun Bun has been hard at work
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:10]:
```suggestion
Trino Fest is drawing ever closer. Commander Bun Bun has been hard at work
```

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 26]
```diff
@@ -0,0 +1,157 @@
...
+<div class="card-deck spacer-30">
+    <a class="btn btn-orange" href="http://www.starburst.io/info/trino-fest-2024?utm_medium=trino&utm_source=website&utm_campaign=Global-FY25-Q2-EV-Trino-Fest-2024&utm_content=Blog-2">
+        Register to attend!
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+Still on the fence, though? Read on for a preview of our speaker lineup and
+brief summaries of their talks. Companies being represented include names like
+Apple and Amazon, so you know it's going to be good.
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:30]: Do we hav Amazon confirmed already now?

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 27]
```diff
@@ -0,0 +1,157 @@
...
+
+<!--more-->
+
+## A brief word from our sponsors...
+
+Thank you to our current sponsors for making this event happen...
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:36]: Do we want to mention we still want more sponsors?

Comment (mosabua) [reply _posts/2024-04-15-trino-fest-2024-approaches.md:36]: Or drop "current"

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 28]
```diff
@@ -0,0 +1,157 @@
...
+major use cases in the past few years. We're excited to say that at Trino Fest,
+we'll have representation for each of the modern big three table formats:
+Iceberg, Delta Lake, and Hudi.
+
+### Iceberg
+
+[Apache Iceberg](https://iceberg.apache.org/) will be covered twice: Amogh
+Jahagirdar from [Tabular](https://tabular.io/) will be diving into the world of
+Iceberg views and how they can be leveraged to coordinate across different query
+languages and dialects. Amit Gilad from Cloudinary will be covering the story of
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:75]: Link ?

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 29]
```diff
@@ -0,0 +1,157 @@
...
+migrating out of Snowflake to the wonderful world of open table formats and
+Iceberg.
+
+### Delta Lake
+
+Marius Grama, a Trino contributor at [Starburst](https://www.starburst.io/),
+will be going into detail on the history, development, and improvements to the
+[Delta Lake](https://delta.io/) connector. With time travel for the Delta Lake connector landed in
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:83]: link to blog post?

Comment (mosabua) [reply _posts/2024-04-15-trino-fest-2024-approaches.md:83]: Also .. landing in 445?

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 30]
```diff
@@ -0,0 +1,157 @@
...
+Trino 445, it's one of the most exciting areas for development in open source
+Trino, and there's some interesting stories that Marius is excited to share with
+the community.
+
+### Hudi
+
+Rounding out data lakes, Ethan Guo from [Onehouse](https://www.onehouse.ai/)
+will be diving into Trino's[Hudi](https://hudi.apache.org/) connector, giving an
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:91]:
```suggestion
will be diving into Trino's [Hudi](https://hudi.apache.org/) connector, giving an
```

File: _posts/2024-04-15-trino-fest-2024-approaches.md [hunk 31]
```diff
@@ -0,0 +1,157 @@
...
+update on what's landed lately to improve performance and functionality. He'll
+also give a preview of what's coming soon. The features are flying in, and if
+you're a current or prospective user of Hudi with Trino, you won't want to miss
+out.
+
+### Amazon S3
+
+And for all your data lake needs, Dai Ozaki from Amazon will be discussing best
```
Comment (mosabua) [inline _posts/2024-04-15-trino-fest-2024-approaches.md:99]: Is this confirmed?

**Review by mosabua — CHANGES_REQUESTED** [review]
A couple of smaller things

**Review by mosabua — APPROVED** [review]
Lets ship this! Go!

**Pull request merged**